docs/csharp/how-to/compare-strings.md

```diff
@@ -0,0 +1,86 @@
+---
+title: "How to: Compare strings - C# Guide"
+description: Learn how to compare and order string values, with or without case, with or without culture specific ordering
+ms.date: 03/13/2018
+ms.prod: .net
+ms.technology: 
+  - "devlang-csharp"
+ms.topic: "article"
+helpviewer_keywords: 
+  - "strings [C#], comparison"
+  - "comparing strings [C#]"
+author: "BillWagner"
+ms.author: "wiwagn"
+---
+# How to compare strings in C# #
+
+When you compare strings, you define an order among them. Comparisons are used to sort a sequence of strings. Once the sequence is in a known order, it is easier to search, both for software and for humans. Other comparisons may check if strings are the same. These sameness checks are similar to equality, but some differences, such as case differences may be ignored.
+
+The methods that compare strings indicate that one string is greater than or less than the other, or that the two strings are equal. The comparison result depends on rules that specify performing an *ordinal comparison* or a *culture-sensitive comparison* and if case should be considered. You pick the rules that match your scenario.
```

Comment:
What about variant: *one string precedes or follows the other* instead of *is greater than or less than* (I guess, swapping relations is fine). I'm inspired by phrasing in [`IComparable` reference](https://docs.microsoft.com/en-us/dotnet/api/system.icomparable-1.compareto?view=netframework-4.7.1)